std/special/compiler_rt/muloti4_test.zig

```diff
@@ -0,0 +1,76 @@
+const __muloti4 = @import("muloti4.zig").__muloti4;
+const assert = @import("std").debug.assert;
+
+fn test__muloti4(a: i128, b: i128, expected: i128, expected_overflow: c_int) void {
+    var overflow: c_int = undefined;
+    const x = __muloti4(a, b, &overflow);
+    assert(overflow == expected_overflow and (overflow != 0 or x == expected));
```

Comment:
A double-check that this is equivalent would be nice, can't think right now.
https://github.com/llvm-mirror/compiler-rt/blob/9008dc9a4e6c3ac1eb7cdd910c8f74099420d911/test/builtins/Unit/muloti4_test.c

Reply:
I would have thought: `overflow == expected_overflow and (expected_overflow == 0 or x == expected)`

(change the 2nd `overflow` to `expected_overflow`)